pmdarima/arima/seasonality.py

```diff
@@ -59,28 +57,27 @@ def decompose(x, type_, m, filter_=None):
     Returns
     -------
     decomposed_tuple : namedtuple
-        A named tuple with `x`, `trend`, `seasonal`, and `random` components
-        where `x` is the input signal, `trend` is the overall trend, `seasonal`
-        is the seasonal component, and `random` is the noisy component. The
-        input signal `x` can be mostly reconstructed by the other three
-        components with a number of points missing equal to `f`.
+        A named tuple with ``x``, ``trend``, ``seasonal``, and ``random``
+        components where ``x`` is the input signal, ``trend`` is the overall
+        trend, ``seasonal`` is the seasonal component, and `random` is the
+        noisy component. The input signal ``x`` can be mostly reconstructed by
+        the other three components with a number of points missing equal to
+        ``m``.
```

Comment:
RST needs double '``' for properly rendering code in the doc. Hence this change.